assets-and-images.md

```diff
@@ -206,3 +206,48 @@ level classes, like
 or
 [`ImageCache`](https://docs.flutter.io/flutter/services/ImageCache-class.html),
 you'll also notice parameters related to scale.)
+
+## Platform assets
+
+There will also be occasions to work with assets in the platform projects directly. Below are 2 common cases where assets are used before the Flutter framework is loaded and running.
+
+### Updating the app icon
+Updating your Flutter application's launch icon works the same way as updating launch icons in native Android or iOS applications.
+
+![Launch icon](/images/assets-and-images/icon.png)
+
+#### Android
+
+In your Flutter project's root directory, navigate to `.../android/app/src/main/res`. The various bitmap resource folders such as `mipmap-hdpi` already contain placeholder images named `ic_launcher.png`. Simply replace them with your desired assets respecting the recommended icon size per screen density as indicated by the [Android Developer Guide](https://developer.android.com/guide/practices/ui_guidelines/icon_design_launcher.html#size).
+
+![Android icon location](/images/assets-and-images/android-icon-path.png)
+
+Note: if you rename the .png files, you must also match the new name in your `AndroidManifest.xml`'s `<application>` tag's `android:icon` attribute.
+
+#### iOS
+
+In your Flutter project's root directory, navigate to `.../ios/Runner/Assets.xcassets/AppIcon.appiconset`. The directory already contains placeholder images. Simply replace them with the appropriately sized images as indicated by their filename as dictated by the Apple [Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/graphics/app-icon). Keep the original file names.
+
+![iOS icon location](/images/assets-and-images/ios-icon-path.png)
+
+## Updating the launch screen
+
+<p align="center">
+  <img src="images/assets-and-images/launch-screen.png" alt="Launch screen" />
+</p>
+
+Flutter also uses native platform mechanisms to draw transitional launch screens to your Flutter app while the Flutter framework loads. This launch screen will persist until Flutter renders the first frame of your application.
+
+This also implies if you don't call [runApp](https://docs.flutter.io/flutter/widgets/runApp.html) in the `void main()` function of your app, the launch screen will persist forever.
```

Comment:
Maybe after the reference to runApp put in parentheses `(or more specifically, if you don't call [window.render] in response to [window.onDrawFrame] being invoked)`, since you don't actually have to call runApp per se.